### PR TITLE
export image with catalog overlay; add pix0 and pix1 tooltip

### PR DESCRIPTION
**Pull request by panchyo0**
issue #899 and #888

### CODE REVIEW

File: src/components/ImageView/ImageViewComponent.tsx [hunk 1]
```diff
@@ -38,6 +39,11 @@ export const getImageCanvas = (padding: Padding, backgroundColor: string = "rgba
         regionCanvas = regionQuery[0] as HTMLCanvasElement;
     }
 
+    const catalogQuery = $(".catalog-plotly").children().children().children(".gl-container").children(".gl-canvas-context");
```
Comment (veggiesaurus) [inline src/components/ImageView/ImageViewComponent.tsx]: should use `?` before each of the `.children()` values just in case

**Review by veggiesaurus — CHANGES_REQUESTED** [review]

File: src/components/ImageView/ImageViewComponent.tsx (hunk 1 above)
Comment (panchyo0) [reply src/components/ImageView/ImageViewComponent.tsx]: Thanks

**Review by veggiesaurus — APPROVED** [review]
looks fine to me. @kswang1029 please check. We can keep the blurry image issue constrained to #889

**Comment by kswang1029** [comment]
Hmm, problem when export as png. 
h_m51_i_s05_drz_sci fits-image-2020-7-22-9-14-28

On screen I see
Screen Shot 2020-07-22 at 9 14 39 AM

It feels like the deviceRatio issue? The above png is generated with a _non-retina_ display (iMac27 2K screen).

**Comment by veggiesaurus** [comment]
can confirm latest commit fixes issue :+1:

**Pull request merged**